### ISSUE TITLE
Need a way to specify container volume

### ISSUE THREAD

**Issue by bbartley** [Bioprotocols/container-ontology#26]
I was perusing the properties in Protege and I couldn't find anything like a `hasVolume` property (although, it seems like I remember seeing a `wellVolume` property at some point).  A `hasVolume` property would be generally useful, as many container types come in different volumes (e.g., standard sizes for microfuge tubes are 0.5, 1.5, 2.0 mL, standard sizes for conical tubes are 15 mL and 50 mL).  So this property will probably come in handy for more precisely specifying common containers.

**Comment by rpgoldman** [comment]
You are right, for microplates there is only a `wellVolume` property because of the ambiguity of using `hasVolume` for a multi-well plate: total volume or single well volume?  And so far, I really only have microplates.

If you have an example of some containers that are *not* microplates, that would help me make the extension, and would generally improve the ontology.

If you have pointers to data sheets or other specs, that would be *really* great. I will try to translate anything you can give me.

**Comment by photocyte** [comment]
> If you have an example of some containers that are not microplates, that would help me make the extension, and would generally improve the ontology.

> If you have pointers to data sheets or other specs, that would be really great. I will try to translate anything you can give me.

Hi there, a quick overview of the most commonly used tubes/vials in bioscience relevant to automation can be seen here:
https://shop.opentrons.com/consumables/ 

(Other automation vendors might have similar "shortened lists" of automation relevant containers). Definitely the "space" to classify all non-microplate containers into a sensible ontology, might be a bit unexplored. Such non-microplate containers often aren't even mentioned explicitly in biological protocols - as there is just an assumption that people will use whatever 10-100 mL esq container that they might have lying around.

Usually, for these types of containers, there aren't technical data sheets like microplates might have. For something like an Opentrons OT2 instrument, the containers fit into these fairly large holders:
https://shop.opentrons.com/4-in-1-tube-rack-set/ 
https://shop.opentrons.com/aluminum-block-set/

And since the mechanical tolerances / where the pipette needs to be is pretty forgiving, not much in way of data sheets or precise calibration.
